### PR TITLE
Client: fix build error on FreeBSD (from Jason Darfus)

### PR DESCRIPTION
**Pull request by davidpanderson**

### CODE REVIEW

File: client/client_state.cpp [hunk 1]
```diff
@@ -431,7 +431,7 @@ static void set_client_priority() {
 #ifdef __linux__
     char buf[1024];
     snprintf(buf, sizeof(buf), "ionice -c 3 -p %d", getpid());
-    system(buf);
+    if (!system(buf)) {}
```
Comment (AenBleidd) [inline client/client_state.cpp:434]: What's the purpose of this change? It basically changes nothing until we want to put smth between curly braces

Comment (davidpanderson) [reply client/client_state.cpp:434]: It removes a compiler warning
